<?xml version="1.0" encoding="UTF-8"?>
<!--
 Copyright 2022 Petr MrÃ¡zek

 This source is subject to the Microsoft Permissive License (MS-PL).
 Please see the COPYING.md file for more information.
-->
<ui version="4.0">
 <class>ImportFTB::FTBAPage</class>
 <widget class="QWidget" name="ImportFTB::FTBAPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1461</width>
    <height>1011</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTreeView" name="packList">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="packDescription"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
